src/main/sensors/gyro.c

```diff
@@ -106,6 +110,11 @@ typedef struct gyroSensor_s {
     biquadFilter_t notchFilter2[XYZ_AXIS_COUNT];
     filterApplyFnPtr notchFilterDynApplyFn;
     biquadFilter_t notchFilterDyn[XYZ_AXIS_COUNT];
+    timeUs_t overflowTimeUs;
+// Fix to make F1 fit into flash in 3.2
+#if (FLASH_SIZE > 128)
```

Comment:
This shouldn't be required, since it shouldn't make any difference to the ROM size.